src/service/cluster.py

```diff
@@ -222,12 +218,17 @@ def wait_for_cluster_ready(self, cluster_id: str) -> bool:
     def _get_addon_install_request_file_path(
         self, addon_id: str, params: dict[str, Any]
     ) -> str:
-        body = self._addon_install_data.copy()
```

Comment:
@alfonsomthd
`_addon_install_data` was getting changed after the first call to this, i.e. `_addon_install_data` was saving the additions to `body` itself and on the second call, the data from the first call was also present.